### ISSUE TITLE
[TRACKING] reCAPTCHA and related design changes

### ISSUE THREAD

**Issue by machikoyasuda** [cal-itp/benefits#2537]
This is the design ticket: https://github.com/cal-itp/benefits/issues/2494

Recommended order of dev tickets:
- [x] #2538
- [x] #2539
- [x] #2542 
- [x] #2543
- [x] #2546
- [x] #2544
- [x] #2545
- [x] #2541
- [x] #2597 
- [x] #2540

All these tickets should be completed before a deploy.

All tickets will be merged into https://github.com/cal-itp/benefits/tree/refactor/recaptcha-copy first

**Comment by machikoyasuda** [comment]
@thekaveman @angela-tran @lalver1 

I ended up making tickets for the realignment of each page, as it requires changing every individual template (rather than say, being able to easily change all of this in 1 template file or 1 CSS change), and I wanted to make smaller PRs that are focused and easier to QA. If finishing all of these tickets then reveals to us that greater refactors can be made to template files or CSS, I think that should be a new ticket. The focus of these tickets is to get the app in a clean deployable state.

**Comment by indexing** [comment]
@machikoyasuda Response to your question in the description here: https://github.com/cal-itp/benefits/issues/2494. 

TL;DR Let's add the informational copy everywhere we use reCaptcha.

**Comment by machikoyasuda** [comment]
Woohoo!
